src/apis/emscripten/time.rs

```diff
@@ -155,21 +153,24 @@ pub extern "C" fn _localtime(time_p: u32, instance: &mut Instance) -> c_int {
     // NOTE: emscripten seems to want tzset() called in this function
     //      https://stackoverflow.com/questions/19170721/real-time-awareness-of-timezone-change-in-localtime-vs-localtime-r
 
+    let timespec = unsafe {
+        let mut time_p_addr = instance.memory_offset_addr(0, time_p as _) as *mut i64;
+        let seconds = *time_p_addr.clone();
```

Comment:
Is this the right way to copy the value from a number pointer? I wasn't sure.

Reply:
Yup, is there any reason you think we should clone it?

Maybe something like could be simpler?

```
let seconds = instance.memory_offset_addr(0, time_p as _) as *const i64;
time::Timespec::new(*seconds, time_p_addr)
```

Thoughts?

Reply:
no reason at all to clone. I will try that out!